lib/rules/media-feature-name-no-vendor-prefix/__tests__/index.js

```diff
@@ -13,47 +14,57 @@ testRule({
 		{
 			code: '@media (device-pixel-ratio: 2) {}',
 		},
+		{
+			code: '@media (min-device-pixel-ratio: 2) {}',
+		},
 	],
 
 	reject: [
 		{
 			code: '@media (-webkit-min-device-pixel-ratio: 1) {}',
+			fixed: '@media (min-device-pixel-ratio: 1) {}',
 			message: messages.rejected,
 			line: 1,
 			column: 9,
 		},
 		{
 			code: '@media (-wEbKiT-mIn-DeViCe-PiXeL-rAtIo: 1) {}',
+			fixed: '@media (mIn-DeViCe-PiXeL-rAtIo: 1) {}',
 			message: messages.rejected,
 			line: 1,
 			column: 9,
 		},
 		{
 			code: '@media (-WEBKIT-MIN-DEVICE-PIXEL-RATIO: 1) {}',
+			fixed: '@media (MIN-DEVICE-PIXEL-RATIO: 1) {}',
 			message: messages.rejected,
 			line: 1,
 			column: 9,
 		},
 		{
 			code: '@media\n\t(min--moz-device-pixel-ratio: 1) {}',
+			fixed: '@media\n\t(min-device-pixel-ratio: 1) {}',
 			message: messages.rejected,
 			line: 2,
-			column: 3,
+			column: 6,
```

Comment:
What has caused the column to change?

Reply:
@vankop It is your chance. Please, help

Reply:
regexp `/-[a-z-]+device-pixel-ratio/gi` since it matches instead of `min--moz` matches `--moz`